packages/flutter/lib/src/painting/box_border.dart

```diff
@@ -212,7 +212,8 @@ abstract class BoxBorder extends ShapeBorder {
   ///
   /// See also:
   ///
-  ///  * [paintBorder], which is used if the border is not uniform.
+  ///  * [paintBorder], which is used if the border has non-uniform colors or styles and no borderRadius.
+  ///  * [Border.paint], which is the similar method in [Border] with additional comments.
```

Comment:
What do you mean by "additional comments"? More context? I'd reword this in any case.

Reply:
Yes. I'll reword. BorderDirectional is shorter and gives less detail on what each parameter do than the equivalent Border.

Reply:
Done. Replacement is:

```dart
[Border.paint], similar to this method, includes additional comments and provides more detailed information on each parameter than described here.
```